src/coreclr/jit/unwind.h

```diff
@@ -79,32 +79,6 @@ class UnwindBase
     {
     }
 
-// TODO: How do we get the ability to access uwiComp without error on Clang?
-#if defined(DEBUG) && !defined(__GNUC__)
-
-    template <typename T>
-    T dspPtr(T p)
-    {
-        return uwiComp->dspPtr(p);
-    }
-
-    template <typename T>
-    T dspOffset(T o)
-    {
-        return uwiComp->dspOffset(o);
-    }
-
-    static const char* dspBool(bool b)
-    {
-        return (b) ? "true" : "false";
-    }
```

Comment:
No need to define these here as far as I can tell, the places that use these functions can just access them from jit.h and utils.h directly.